Source/WebGPU/WGSL/tests/valid/overload.wgsl

```diff
@@ -99,3 +99,32 @@ fn testTextureSample() {
     let r = textureSample(t, s, vec3<f32>(0, 0, 0), 0);
   }
 }
+
+fn testVec2() {
```

Comment:
I still don't quite understand how our testing strategy for WGSL works right now...

Reply:
These are only validating type checking. All these expressions would fail to type check without this patch. I described it in a different PR, but for now unfortunately this can't be automated, but as soon as I land the standalone compiler and sort a couple other things we can automate these.